docs/old-reference-guide/modules/ROOT/pages/known-issues-and-workarounds.adoc

```diff
@@ -0,0 +1,55 @@
+= Known Issues and Workarounds
+:navtitle: Known Issues and Workarounds
+
+As with tool, library, or framework, there are well-known issues you might potentially come up against.
+Especially given the fact Axon Framework will integrate with a wide variety of other tools, it becomes rough to deal with everything.
+
+This page intends to provide the biggest "gotchas," with optional workarounds, which you can see below.
+
+[NOTE]
+.Where is issue XYZ?
+====
+This page can by no means contain **all** possible issues out there.
+However, that does not mean we are not open for suggestions.
+
+This documentation, just as the framework, is open source as well.
+Hence, if you have found something problematic that you think should be added to the list, be sure to file an issue and (ideally!) a pull request.
+
+You can make an enhancement request by clicking link:https://github.com/AxonFramework/AxonFramework/issues/new?assignees=&labels=Type%3A+Enhancement&projects=&template=2_enhancement_request.md[here].
+When looking to provide a pull request, be sure to adjust link:https://github.com/AxonFramework/AxonFramework/blob/master/docs/old-reference-guide/modules/ROOT/pages/serialization.adochttps://github.com/AxonFramework/AxonFramework/blob/master/docs/old-reference-guide/modules/ROOT/pages/known-issues-and-workarounds.adoc[this file].
+====
+
+== Relation database storage and serialized data columns
+
+When storing the serialized formats of, for example, xref:axon-framework-reference:events:event-processors/streaming.adoc#tracking-tokens[tokens], xref:axon-framework-reference:sagas:index.adoc[sagas], xref:axon-framework-reference:tuning:event-snapshots.adoc[snapshots], or xref:axon-framework-reference:events:infrastructure.adoc[events], it is good to know Axon uses the (B)LOB for these columns in your RDBMS of choice.
+
+Some databases react unexpectedly to this, with a clear case being link:https://www.postgresql.org/[PostgreSQL].
+It uses a mechanism it calls link:https://wiki.postgresql.org/wiki/TOAST["TOAST"] to move data that's too large to another table.
```

Comment:
I might be misreading the sources, but it looks like the blog's title is misleading.

TOAST is actually transparent for the user, and more powerful. This is automatically used for bytea (byte array) data types. For LOB datatypes, the OID approach is used, which is quite annoying, as it requires cleanup.
The Postgres driver for Hibernate maps `@Lob` annotated fields to LOB data types.

So we want more TOAST, not less.

Reply:
It's both TOAST. However, one is eager and the other is lazy.

Hibernate maps to LOB columns automatically, which is eagerly TOASTing. This means that all token values are stored into the TOAST table of the token_entry table. 
Now, you can't really clean up the OID table (which we experienced first-hand in Inspector). In addition, it needs an extra read.
As tokens generally far from exceed 8KB (especially compressed), there's not reason to toast at all. Setting the datatype to bytea (lazy toasting) is thus basically leading to never toast :) 

For context, as I did some digging for inspector; To clean up OIDs, you have a special command. This makes a copy of all the OID values that still have a reference, reordens it, and then deleted all the values not in that table in the original table. It's horrible to manage :sweat:

Reply:
I have decided it would be better to simply add those changes to this PR. So, I did just that right now.

Hence, @abuijze and @CodeDrivenMitch, please review it again. Your opinion is much appreciated in this area.